thoth/storages/graph/models.py

```diff
@@ -1471,6 +1471,24 @@ class PythonPackageMetadataDistutils(Base, BaseExtension):
     python_packages_metadata = relationship("HasMetadataDistutils", back_populates="python_package_metadata_distutils")
 
 
+class KebechetGithubAppInstalltions(Base, BaseExtension):
+    """Kebechet Github App Installations.
+
+    slug = namespace/repository (ex - thoth-station/advisor)
+    repo_name = repository (ex - advisor)
+    private = True or False
+    installation_id = proviced by github (ex - 236821515)
+    """
+
+    __tablename__ = "kebechet_github_installations"
+
+    id = Column(Integer, primary_key=True, autoincrement=True)
+    slug = Column(Text, nullable=False, unique=True)
+    repo_name = Column(Text, nullable=True)
+    private = Column(Boolean, nullable=True)
```

Comment:
Not nullable here so we have 2 distinct values (private / not private).

BTW do we need to have nullables on other properties (repo name, installation id, ...).

Reply:
We don't need them to be nullable, let's make nullable false on all of these. :)